commands.md

```diff
@@ -24,12 +24,14 @@ the network simulator commands when writing test files.
 enter <name>            // should be called before any other command dealing with the named peer
 hops <name> <number>    // should be called before starting a peer to have any effect
 caps <name> <string>    // should be called before starting a peer to have any effect
+nolog <name>            // should be called before starting a peer to have any effect (omits log.offset when loading identity from fixtures)
```

Comment:
maybe rename this to `dontseedmsgs` ? I feel like brevity is confusing with this one.

Reply:
i'll keep that in mind and try to find something better!

Reply:
trying out `skipoffset` for now :)